IAB Workshop.md

```diff
@@ -1,25 +1,25 @@
-IAB workshop on the Next Era of Network Management Operations (NEMOPS)
+IAB workshop on the Next Era of Network Management Operations (NEMOPSWS)
```

Comment:
```suggestion
IAB workshop on the Next Era of Network Management Operations (NEMOPS)
```

I prefer to not have WS in the name. This is also aligned with the naming used in the past: https://www.iab.org/workshops/past-workshops/

Reply:
If this is a title, then no need to add WS. But note that the WS will show up on the data tracker page URL and the name of the group there....